feature/contributor/src/main/java/io/github/droidkaigi/confsched2020/contributor/ui/viewmodel/ContributorsViewModel.kt

```diff
@@ -1,11 +1,87 @@
 package io.github.droidkaigi.confsched2020.contributor.ui.viewmodel
 
+import androidx.lifecycle.LiveData
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.asLiveData
+import androidx.lifecycle.liveData
+import io.github.droidkaigi.confsched2020.ext.combine
+import io.github.droidkaigi.confsched2020.ext.toAppError
+import io.github.droidkaigi.confsched2020.ext.toLoadingState
+import io.github.droidkaigi.confsched2020.model.AppError
+import io.github.droidkaigi.confsched2020.model.Contributor
+import io.github.droidkaigi.confsched2020.model.LoadState
 import io.github.droidkaigi.confsched2020.model.repository.ContributorRepository
+import java.lang.Exception
 import javax.inject.Inject
 
 class ContributorsViewModel @Inject constructor(
     private val contributorRepository: ContributorRepository
 ) : ViewModel() {
-    // TODO Implement
+
+    /**
+     * [ViewState] has to be wrapped to workaround a compile error about type mismatch.
+     * [combine] doesn't seem to understand child classes.
+     */
+    data class UiModel(val viewState: ViewState) {
```

Comment:
I know this deviates a bit from the pattern used in this repo. The challenge I encountered is the following:

When there's no cache yet, the LiveData emits an empty contributors list and then makes an API call to fetch the list. I need to keep the loading indicator visible since an API call technically could take a noticeable amount of time depending on the user's network condition. 

Long story short, I first tried to follow the existing UiModel pattern, but then IMO things started to become harder to understand as I have to think about more cases/states of `contributorsLoadStateLiveData`. e.g. Would have to take more conditions in consideration to decide when `isLoading` should be true or false.

Having said that, if you still prefer the existing pattern with UiModel having `isLoading`, `error`, and `contributors`, I can try to get that to work properly 👍

Reply:
sorry. I did not read this. I'll check it. 🙏

Reply:
Thanks for checking in in various ways. 👍 
> Having said that, if you still prefer the existing pattern with UiModel having isLoading, error, and contributors, I can try to get that to work properly 👍

There may be some difficulties, but we want to check the validity of UiModel.